wiki/SourceCode.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Source Code
+Title: Source Code
```

Comment:
This used to be ``title`` (lower case), and should probably stay in lower case - although I didn't immediately find anything on the Jekyll website saying that explicitly - see e.g. https://jekyllrb.com/docs/front-matter/